sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/openshift/OpenShiftPostgresqlIT.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.ts.reactive.openshift;
+
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.ts.reactive.PostgresqlDatabaseIT;
+
+@OpenShiftScenario
+//TODO: enable disabled methods then the issue will be fixed
+public class OpenShiftPostgresqlIT extends PostgresqlDatabaseIT {
+    @Override
```

Comment:
https://github.com/quarkusio/quarkus/issues/21700 seems to be fixed in 2.6.2.Final by https://github.com/quarkusio/quarkus/pull/22644

Please check if the tests are passing now and create new PR for drop of  `@Disabled("https://github.com/quarkusio/quarkus/issues/21700")`

Reply:
Will do
https://trello.com/c/9mpNtShc/1320-enable-disabled-tests-in-hibernate-reactive